docs/kubernetes/kubernetes-production-google-cloud.md

```diff
@@ -0,0 +1,93 @@
+# Production on Google Cloud Platform using Kubernetes, Terraform and Kustomize
+
+Shopsys Framework provides a way how to deploy your site on Google Cloud using Kubernetes manifests, Terraform configurations and Kustomize written into repository.
+
+We created shell scripts that encapsulate functionality of Terraform and Kustomize on google-cloud. This documents only describes functionality of each technology.
+
+If you want to know about the script usage, read [Deploy to Google Cloud from CI](deploy-to-google-cloud-from-ci.md).
+
+## Google Cloud
+Create a user on Google Cloud.
+
+After creating user create project.
+
+### Obtain Parameters and service-account.json
+
+1. service-account.json
+
+    Read more here https://cloud.google.com/iam/docs/creating-managing-service-account-keys
+
+2. Project id
+
+3. Your Login to Google Cloud
+
+### Allow API
+Allow API so you can control resources and providers using terraform.
```

Comment:
```suggestion
Enable `kubernetes engine API` so you can control resources and providers using terraform.
```

Reply:
or we can provide image from our working example 
![image](https://user-images.githubusercontent.com/39240194/49294075-324c0900-f4b2-11e8-8cab-bb72fd61046d.png)

Reply:
or we can get it from usage metrics
![image](https://user-images.githubusercontent.com/39240194/49339361-ce9c1a00-f630-11e8-8584-f6f8e6bcd351.png)